specification/ml/get_job_stats/MlGetJobStatsRequest.ts

```diff
@@ -21,15 +21,39 @@ import { RequestBase } from '@_types/Base'
 import { Id } from '@_types/common'
 
 /**
+ * Retrieves usage information for anomaly detection jobs.
+ * You can get statistics for multiple anomaly detection jobs in a single API
```

Comment:
> You can get statistics for multiple anomaly detection jobs in a single API...

IMO this detail doesn't need too much prominence. In https://github.com/elastic/elasticsearch-specification/pull/852 I therefore moved that info into the appropriate parameter description:
https://github.com/elastic/elasticsearch-specification/blob/3149ef4eade38ebcd953e3c74b5bdbf6854be711/specification/ml/get_calendars/MlGetCalendarsRequest.ts#L34

If you agree, we should do the same for all these type of APIs.

Reply:
Addressed in 8d49755.